repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -188,4 +198,36 @@ object ProjectUpdateFileRequest {
       ProjectUpdateFileRequest.apply,
       unlift(ProjectUpdateFileRequest.unapply)
     )
-}
\ No newline at end of file
+}
+
+/**
+ * Class represents dummy object
+ */
+final case class DummyResponseBody()
+object DummyResponseBody {
+  implicit val dummyResponseBodyNonStrictReads = Reads.pure(DummyResponseBody())
+  implicit val dummyResponseBodyWrites = OWrites[DummyResponseBody](_ => Json.obj())
+}
+
+/**
+ * Wrapper type contains success response message from [[cromwell.pipeline.service.SuccessResponseBody]]
+ *
+ * @param message any response info which signals successful request
+ */
+final case class SuccessResponseMessage(message: String) extends AnyVal
+
+object SuccessResponseMessage {
+  implicit lazy val successResponseMessageFormat: OFormat[SuccessResponseMessage] =
+    Json.format[SuccessResponseMessage]
+}
+
+/**
+ * Wrapper type contains empty response message to use in [[cromwell.pipeline.service.HttpClient]]
+ *
+ * @param message empty payload message
+ */
+final case class EmptyPayload private (message: String = "")
```

Comment:
It makes no sense to create a `case class` with a private constructor. Anyone can create an instance using `EmptyPayload.apply` method.

Reply:
done, modifier removed